### ISSUE TITLE
[Marker] How to fix icon size regardless of the zoom in/out?

### ISSUE THREAD

**Issue by banlife-tykim** [react-native-maps/react-native-maps#4270]
# Feature request
Can i fix custom marker image size regardless of the zoom in/out? 

## Possible implementation
"expo": "^44.0.0",
"react-native-maps": "^0.30.2"

### Code sample
![스크린샷 2022-06-13 오후 2 57 24](https://user-images.githubusercontent.com/86279461/173289147-d7829e86-a3f7-45f3-8633-bf61292c4592.png)

**Comment by Simon-TechForm** [comment]
I don't believe that this is a feature request? :) I suggest taking questions like this to stackoverflow, as we want to keep the issue section focused on issues and feature requests :)
